.github/dependabot.yml

```diff
@@ -3,7 +3,8 @@ updates:
   - package-ecosystem: gomod
     directory: /
     schedule:
-      interval: daily
+      interval: weekly
```

Comment:
I'm a bit worried that we might get flooded by these updates on Mondays (or Fridays, in case of https://github.com/cilium/cilium/pull/22176/files#r1022501474).

IMO the scarce resource here is developer's review time (especially given that the @cilium/vendor team consists of only 3 members) and personally I prefer having 1-2 dependabot reviews every other day rather than a whole batch of them at the end/beginning of the week. But of course other folks' preferences might differ.

Reply:
What needs to be reviewed? Could we extend the review team? When I reviewed those PRs, I usually only looked at whether the CI was passing...

Reply:
Same question as Paul. Can these reviewed by tophats?

Reply:
another option would be to assign these pull requests to @cilium/tophat instead of @cilium/vendor. then for each tophat rotation we can do one round of reviews on friday and be done with it.

Reply:
For Go module updates, I usually check the following:

1. Check whether the Go-related GH actions (namely the module vendoring check) passed.
2. Check whether there are any vendored changes. If yes, trigger CI. If no, approve and mark as ready to merge.
3. Check whether any of our forked modules for the `replace` directive in `go.mod` require an update as a consequence.

It's usually 1-2 min per PR, so not a lot of effort. It's something that I usually do e.g. when waiting for a build or tests to run.

Reply:
> For Go module updates, I usually check the following:
> 
> 1. Check whether the Go-related GH actions (namely the module vendoring check) passed.
> 2. Check whether there are any vendored changes. If yes, trigger CI. If no, approve and mark as ready to merge.
> 3. Check whether any of our forked modules for the `replace` directive in `go.mod` require an update as a consequence.

Oh, forgot one important point. In cases there are vendored changes, I usually check the upstream release notes/logs whether they mention any breaking changes that might affect us.

Reply:
Another thing I do: whenever a new dep is added, I check that the license is compatible and that the dep seems used and well maintained.